pages/content/amp-dev/documentation/guides-and-tutorials/develop/interactivity_guide/index.md

```diff
@@ -0,0 +1,104 @@
+
+Building interactive experiences in AMP may feel untraditional, and it is! But that doesn’t mean it’s more difficult. Many interactive experiences are very easy to build with AMP once you’re introduced to concepts and understand the syntax.
+
+There are several levels of interactivity in AMP. You can apply one, multiple, all, or none to an AMP page. Read the sections of this guide to learn how to build interactivity in AMP.
+
+# Interactivity foundations
+
+This guide outlines the built-in interactivity AMP provides. It explains how to easily implement common primitives, such as hiding and showing elements.
+
+[example preview="inline" playground="true"]
+```html
+  <h1 id="hello">Hello World!</h1>
+  <button on="tap:hello.hide">Hide</button>
+  <button on="tap:hello.show">Show</button>
+  <button on="tap:hello.toggleVisibility">Toggle</button>
+```
+[/example]
+
+[Start here](foundations.md) to learn foundations that you can build and combine with other interactive possibilities.
+
+# Ready-made interactivity with AMP components
+
+One of AMP’s greatest strengths is its extensive library of ready-to-go components. Many of them are interactive elements that you can customize and combine to build unique experiences!
+
+[example preview="top-frame" playground="true" orientation="portrait" imports="amp-sidebar, amp-accordion"]
+```html
+  <amp-sidebar id="sidebar" class="sample-sidebar" layout="nodisplay" side="right">
+    <h3>Sidebar</h3>
+    <amp-accordion id="my-accordion" disable-session-states>
+      <section>
+        <h2>Section 1</h2>
+        <p>Content in section 1.</p>
+      </section>
+      <section>
+        <h2>Section 2</h2>
+        <div>Content in section 2.</div>
+      </section>
+      <section expanded>
+        <h2>Section 3</h2>
+        <amp-img src="/static/inline-examples/images/squirrel.jpg" width="320" height="256"></amp-img>
+      </section>
+    </amp-accordion>
+    <button on="tap:sidebar.close">Close sidebar</button>
+    <button on="tap:sidebar.toggle">Toggle sidebar</button>
+  </amp-sidebar>
+  <button on="tap:sidebar.toggle">Toggle sidebar</button>
+  <button on="tap:sidebar.open">Open sidebar</button>
+```
+[/example]
+
+[This guide](ready_made.md) introduces the most common interactive components and outlines implementation patterns.
+
+# Building custom interactive experiences
+
+While AMP provides solutions to common web widgets and interactivity, each website has its own needs. AMP embraces highly customizable experiences with three components: [`amp-selector`](../../../components/reference/amp-selector.md), [`amp-bind`](../../../components/reference/amp-bind.md), and [`amp-script`](../../../components/reference/amp-bind.md).
```

Comment:
To me `amp-selector` is not obviously in the same category as `amp-bind` and `amp-script`, maybe elaborate more on why these make sense to group together?

Reply:
added a bit more information, hopefully that helps!